duckduckgo/macos-browser
# Improve Asana integration for failed PR checks

Task/Issue URL: https://app.asana.com/0/1203301625297703/1205530774734473/f

**Description**:

**Steps to test this PR**:
See [this Asana task](https://app.asana.com/0/1205237866452338/1205566966703322/f) for an example run of the new integration.

If you want to test it yourself:
1. Revert https://github.com/duckduckgo/macos-browser/pull/1673/commits/d7871c68699167b801c1491234ea85a02d238941 commit.
2. Push to this branch.
3. Wait for the task to be created (and assigned to you).
4. Go to the workflow and have it rerun failed checks.
5. The workflow will still fail but it will close Asana task, because that's how the https://github.com/duckduckgo/macos-browser/pull/1673/commits/d7871c68699167b801c1491234ea85a02d238941 commit makes it do.
6. Revert your most recent commit to restore production-ready state.




---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
